src/js/media-time-display.js

```diff
@@ -52,6 +52,12 @@ const updateAriaValueText = (el) => {
 };
 
 class MediaTimeDisplay extends MediaTextDisplay {
+  constructor() {
+    super();
+
+    this.shadowRoot.styleSheets[0].addRule(':host', 'white-space: nowrap');
```

Comment:
the way that the elements are currently set up in media-chrome, there isn't a way to add extra css to be rendered into the template in the constructor.

This adds an extra css rule after media-text-display has rendered itself.

Reply:
While this works, &nbsp; is simpler still.